docs/environment-variables.md

```diff
@@ -43,8 +43,9 @@ those.
 
 - `GRAPH_MAPPING_HANDLER_TIMEOUT`: amount of time a mapping handler is allowed to
   take (in seconds, default is unlimited)
-- `GRAPH_IPFS_TIMEOUT`: timeout for ipfs requests. In seconds, default is 60.
-  seconds.
+- `GRAPH_IPFS_SUBGRAPH_LOADING_TIMEOUT`: timeout for IPFS requests made to load
+  subgraph files from IPFS (in seconds, default is 60).
+- `GRAPH_IPFS_TIMEOUT`: timeout for IPFS requests (in seconds, default is 60).
```

Comment:
Could we say something like 'timeout for IPFS requests from mapping functions (like ipfs.cat and ipfs.map)` to make it clearer that one is about the internal workings of graph-node and the other is what people are more likely to want to influence?

Reply:
Yep, can do.

Reply:
Updated.

Reply:
Great! Thanks!